Revit_oM/Elements/DraftingInstance.cs

```diff
@@ -20,19 +20,20 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
+using BH.oM.Adapters.Revit.Interface;
 using BH.oM.Adapters.Revit.Properties;
 using BH.oM.Base;
 using BH.oM.Geometry;
 
 namespace BH.oM.Adapters.Revit.Elements
 {
-    public class DraftingObject : BHoMObject
+    public class DraftingInstance : BHoMObject, IInstance 
     {
         /***************************************************/
         /**** Public Properties                        ****/
         /***************************************************/
 
-        public ObjectProperties ObjectProperties = new ObjectProperties();
+        public InstanceProperties InstanceProperties { get; set; } = new InstanceProperties();
```

Comment:
Agreeing on changing the name for this away from the very generic `ObejctProperties`, but `InstanceProperties` is almost as generic. Would something like `DraftingProperties` not be clearer for the name of the class? Might be wrong here though if this is used for multiple things

Also I would call the name of the property just properties, as you already know what kind of object you are using. Something like:

```
public DraftingProperties Properties { get; set; } = new DraftingProperties ();
```

I think would make more sense

Reply:
Correct.

Reply:
Just to add on that. I always assume inputs should be named in such way user do not have to think to much what should be connected to it... so if input is called "InstanceProperties" you know straight away what kind of input you need to connect... if you start searching for just "Properties" it may appear there is many objects with such a name.